Gemfile

```diff
@@ -51,6 +50,14 @@ gem 'sidekiq-statsd', '0.1.5'
 
 gem 'deprecated_columns', '0.1.0'
 
+if ENV['GDS_API_ADAPTERS_DEV']
```

Comment:
I've left this in for now as it has been useful and it doesn't enforce that anyone use the ENV variable in development.